pkgs/development/tools/godot/4/mono/make-deps.nix

```
@@ -0,0 +1,63 @@
+{ pkgs, nuget-to-nix }:
+
+let godot4-mono = pkgs.callPackage ./default.nix {}; in
+
+godot4-mono.overrideAttrs (self: base: {
+  pname = "godot4-mono-make-deps";
+
+  nativeBuildInputs = base.nativeBuildInputs ++ [ nuget-to-nix ];
+
+  nugetDeps = null;
+  nugetSource = null;
+  nugetConfig = null;
+
+  shouldConfigureNuget = false;
+
+  outputs = [ "out" ];
+  buildPhase = " ";
```

```suggestion
  buildPhase = " ";
```
runHooks are missing. Try:

> dontBuild = true;